config/application.rb

```diff
@@ -23,5 +23,8 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+    config.session_store :cookie_store, key: '_interslice_session'
+    config.middleware.use ActionDispatch::Cookies
+    config.middleware.use config.session_store, config.session_options
```

Comment:
By convention, **_REST APIs_** are stateless, which means that every interaction is independent, 
and each request and response provides all the information required to complete the exchange.
But, in your implementation, it seems you're persisting state. Kindly revert to the stateless option as 
we'll manage the app state with redux locally. 

_**However, you're welcome to give supporting arguments if you think your logic is best suited.**_